packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -590,6 +590,76 @@ rather than using the `TFoot`.
 </Table>`}
 </LivePreview>
 
+### Table with sticky header
+
+Tables support a "sticky header", which can be useful in tables with a large amount of data. A sticky header will allow
+a user to keep the context of what each column contains as they scroll through the data contained in the table.
+
+Set the boolean property `stickyHeader` on the `THead` component, to enable the sticky header.
+
+<LivePreview scope={{Table, THead, TBody, TFoot, Tr, Th, Td, Text, Box}} language="jsx">
+  {`<Box overflowY="auto" maxHeight="500px">
+  <Table tableLayout="fixed">
+    <THead stickyHeader top={0}>
+      <Tr>
+        <Th>Agents</Th>
+        <Th>Skills</Th>
+        <Th textAlign="right">Tasks</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr>
+        <Th scope="row">Adam Brown</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">35</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Adriana Lovelock</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">28</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Amanda Cutlack</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">7</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Sofie Korneliya</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">47</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Gerarda Che</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">2</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Natalia Hiroshi</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">9</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Gaël Kambiz</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">18</Td>
+      </Tr>
+      <Tr>
+        <Th scope="row">Amelina Muriel</Th>
+        <Td>English, French, Sales, Spanish</Td>
+        <Td textAlign="right">29</Td>
+      </Tr>
+    </TBody>
+    <TFoot>
+      <Tr>
+        <Th scope="row">Total tasks</Th>
+        <Td/>
+        <Td textAlign="right">70</Td>
+      </Tr>
+    </TFoot>
+  </Table>
+</Box>`}
+</LivePreview>
```

Comment:
suggestion: the website example would be more realistic for the use case with more rows. For reference, right now the example only scrolls down two rows.